readme.txt

```diff
@@ -99,80 +99,37 @@ There are opportunities for developers at all levels to contribute. [Learn more
 
 == Changelog ==
 
-= 7.1.1 =
+= 7.2 =
 
-* Release date: March 06, 2019
-* Release post: https://wp.me/p1moTy-foJ
-
-**Bug fixes**
-
-* General: avoid conflicting with other plugins when suggesting Jetpack features on the Plugins screen.
-* Publicize: avoid errors when the feature is not active on a site.
-* Widgets: improve performance of the Top Posts and the Blog Stats widgets on high-traffic sites.
-* Subscriptions: fix an issue that prevented displaying subscribers count in the subscription forms.
-* Tiled Galleries / Slideshows: ensure they can be displayed properly in Internet Explorer 11.
-
-= 7.1 =
-
-* Release date: March 05, 2019
-* Release post: https://wp.me/p1moTy-e9x
+* Release date: April 02, 2019
+* Release post: https://wp.me/p1moTy-foe
 
 **Major Enhancements**
 
-* Block Editor: this release introduces 6 new blocks:
-	- the Ads block allows you to insert different ads from [our WordAds program](https://jetpack.com/support/ads/) within your posts and pages.
-	- the Mailchimp block allows your readers to easily subscribe to your Mailchimp newsletter.
-	- the Video block supports VideoPress videos if you've purchased our Premium or Professional plan.
-	- the Slideshow block allows you to insert beautiful slideshows in your posts and pages.
-	- The Business Hours blocks is useful for companies who want to display their business's Opening Hours on their site.
-	- The Contact Info block is useful for any business who may want to display useful information on a post or page.
+* Adds a Repeat Visitor block that controls block visibility based on how often a visitor has viewed the page.
+* Selectively disable Ads blocks for mobile devices.
```

Comment:
I would expand on that to make it clear we are adding a new option.

Reply:
👍

Reply:
fb97ab5ae